cdap-app-templates/cdap-etl/cdap-etl-realtime/src/main/java/co/cask/cdap/templates/etl/realtime/RealtimeStageContext.java

```diff
@@ -31,7 +27,8 @@
 public class RealtimeStageContext implements StageContext {
```

Comment:
Yes its for Transform. Renaming of Transform related classes will happen in a separate PR.